spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -407,6 +408,34 @@ object TypeParser {
                                         isArrayElement: Boolean)
                                        (implicit defaults: Defaults) extends ScalarParser[String]
 
+  private final case class BinaryParser(field: TypedStructField,
+                                        path: String,
+                                        column: Column,
+                                        origType: DataType,
+                                        failOnInputNotPerSchema: Boolean,
+                                        isArrayElement: Boolean)
+                                       (implicit defaults: Defaults) extends PrimitiveParser[Array[Byte]] {
+    override protected def assemblePrimitiveCastLogic: Column = {
+      origType match {
+        case BinaryType => column.cast(BinaryType) // binary: just cast
+        case StringType =>
+          if (field.structField.metadata.contains(MetadataKeys.Encoding)) {
+            val encoding = field.structField.metadata.getString(MetadataKeys.Encoding)
+            encoding.toLowerCase match {
+              case "base64" => org.apache.spark.sql.functions.unbase64(column).cast(BinaryType)
+              case _ => throw new IllegalStateException(s"Unsupported encoding for Binary field ${field.structField.name}: '$encoding'")
```

Comment:
No option for "leave as it is'.

Reply:
The option to leave 'as-is' is a couple of lines below - omitting the `encoding` metadata key altogether. Do you think there should be another option within `encoding`, too? e.g. `{"encoding":"none"}`?

Reply:
IMHO it would be better, yes.

Reply:
Done.